### ISSUE TITLE
make optionList width configurable

### ISSUE THREAD

**Issue by 75lb** [75lb/command-line-usage#13]
Currently, the width of a `content` section is configurable. Here's an example of a content section with a `maxWidth` of 40: 

```js
const usage = commandLineUsage([
  {
    header: 'A typical app',
    content: {
      options: { maxWidth: 40 },
      data: [
        { col: 'Generates something [italic]{very} important. This is a rather long, but ultimately inconsequential description intended solely to demonstrate description appearance. ' }
      ]
    }
  }
])
```

Output: 

```
A typical app

  Generates something very important.
  This is a rather long, but ultimately
  inconsequential description intended
  solely to demonstrate description
  appearance.
```

However, an `optionList` section currently [has a fixed width of 80 columns](https://github.com/75lb/command-line-usage/blob/master/lib/option-list.js#L41) which is an issue for some users. 

Resolve this issue by making the `maxWidth` of an `optionList` section configurable, the same way as it is for a `content` section.

**Comment by AriLFrankel** [comment]
I'd be happy to work on this :)

**Comment by 75lb** [comment]
Fixed and released in [v5.0.0](https://github.com/75lb/command-line-usage/releases/tag/v5.0.0).
